matrix/common/pandas_utils.py

```diff
@@ -0,0 +1,41 @@
+import pandas
+from pandas.core.frame import DataFrame
+from zarr import Group
+
+
+def convert_dss_zarr_root_to_subset_pandas_dfs(zarr_root: Group, start_row: int, end_row: int):
```

Comment:
thoughts on breaking this into `get_expression_dataframe` and `get_qc_dataframe`?

Reply:
Or - does this currently return a Tuple or just 2 values? Wouldn't be opposed to a Tuple if both dfs are tightly coupled

Reply:
Lets keep this as one for now. As of now, there is no functionality in the matrix service that doesn't interact with both dataframes simultaneously